dockers/texcmp/texcmp.js

```diff
@@ -16,18 +15,12 @@ var readFile = Q.denodeify(fs.readFile);
 var writeFile = Q.denodeify(fs.writeFile);
 var mkdir = Q.denodeify(fs.mkdir);
 
-// ignore some tests, since they contain commands not supported by LaTeX
-var blacklist = {
```

Comment:
I think I'd rather leave this in this file, since it's not related to the other stuff, and gets left in the query parameter?